synapse/rest/admin/rooms.py

```diff
@@ -241,7 +241,9 @@ async def _delete_room(
         if purge:
             await pagination_handler.purge_room(room_id, force=force_purge)
 
-        return 200, ret
+        # Cast safety: I'm casting away the knowledge that this is a TypedDict.
```

Comment:
```suggestion
        # Cast safety: Cast away the knowledge that this is a TypedDict.
```

Reply:
Thanks. I'll also add a link to remind the reader why the cast is necessary in the first place.